test/unit/org/apache/cassandra/db/filter/ANNOptionsTest.java

```diff
@@ -20,6 +20,11 @@
 
 import javax.annotation.Nullable;
 
+import org.apache.cassandra.index.IndexNotAvailableException;
+import org.apache.cassandra.index.sai.SAITester;
+import org.apache.cassandra.index.sai.StorageAttachedIndex;
+import org.apache.cassandra.inject.Injections;
+import org.apache.cassandra.inject.InvokePointBuilder;
```

Comment:
I should check what happened here.... probably unused imports, I was testing some stuff